src/App.jsx

```diff
@@ -14,57 +16,57 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selGood, setSelecGood] = useState(
```

Comment:
The method's name 'setSelecGood' does not start with a verb and may be a typo. Consider renaming it to start with a verb and correct the spelling to 'setSelectedGood' to make it clear what the method does.